components/button/test/index.js

```diff
@@ -93,4 +101,18 @@ describe( 'Button', () => {
 			expect( button.type() ).toBe( 'button' );
 		} );
 	} );
+
+	// Disable reason: This test is desirable, but unsupported by Enzyme in
+	// the current version, as it depends on features new to React in 16.3.0.
+	//
+	// eslint-disable-next-line jest/no-disabled-tests
+	describe.skip( 'ref forwarding', () => {
+		it( 'should enable access to DOM element', () => {
+			const ref = createRef();
+
+			mount( <ButtonWithForwardedRef ref={ ref } /> );
+
+			expect( ref.current.nodeName ).toBe( 'button' );
```

Comment:
This test will fail when unskipped: `Element#nodeName` returns uppercase, so should be asserting as `toBe( 'BUTTON' );`

Reply:
It may take another few weeks until we can unskip those tests :(
The day when we remove `Enzyme` is becoming a reality if there is no progress on supporting React 16.3+